### ISSUE TITLE
Multitenant 

### ISSUE THREAD

**Issue by DavidPDL** [NorskHelsenett/HelseID.Samples#20]
Hi, 

is that possible to use this sample with different client IDs?

**Comment by oyvind-bech** [comment]
Yes, any client ID can be set. See https://utviklerportal.nhn.no/informasjonstjenester/helseid/tilgang-til-helseid/test-token-tjenesten/docs/test-token-tjenesten_enmd/
